metrics/OSI_Approved_Licenses.md

```diff
@@ -30,7 +30,7 @@ Example: http://augur.osshealth.io/repo/Zephyr-RTOS/zephyr/risk
 
 Extract list of licenses from a code base, same as in License Coverage metric.
 Compare the list of licenses to [Licenses.json](https://raw.githubusercontent.com/spdx/license-list-data/master/json/licenses.json) and take note of how many licenses are approved by the OSI.
-Calculate the percentage of files on the OSI license list.
+Calculate the percentage of OSI approved files on the SPDX license list.
```

Comment:
This doesn't make sense nor did the orginal sentence make sense.


Maybe this is better?
```suggestion
Calculate the percentage of files with an on the OSI approved license.
```